applications/StructuralMechanicsApplication/custom_utilities/shellq4_local_coordinate_system.hpp

```diff
@@ -308,7 +308,7 @@ namespace Kratos
 		}
 
 		template<int TComponent>
-		inline const double Coord_ij(int Tid1, int Tid2)const {

```

Comment:
why removing the inline

Reply:
good spotting, this was not intentional!